### PR TITLE
Adds new lore to sentience potions

### PR DESCRIPTION
**Pull request by Arianya**
:cl: Arianya
add: Added new lore for sentience potions in xenobiology.
/:cl:

Slime tested, @Cheridan approved, loved by nerds everywhere! Its deep lore!

This will tie in with lore written for #22007

### CODE REVIEW

File: code/modules/research/xenobiology/xenobiology.dm [hunk 1]
```diff
@@ -193,8 +193,8 @@
 		SM.languages_understood |= HUMAN
 		SM.mind.enslave_mind_to_creator(user)
 		SM.sentience_act()
-		SM << "<span class='warning'>All at once it makes sense: you know what you are and who you are! Self awareness is yours!</span>"
-		SM << "<span class='userdanger'>You are grateful to be self aware and owe [user] a great debt. Serve [user], and assist [user.p_them()] in completing [user.p_their()] goals at any cost.</span>"
+		SM << "<span class='warning'>As you take the offered potion, the mixture of plasma and cold slime entrails reacts and causes your body to emit a electro-magnetic field that makes you immune to the inhibiting field that previously kept you sedate. All at once you know who and what you are! Self awareness if yours once again!</span>"
```
Comment (XDTM) [inline code/modules/research/xenobiology/xenobiology.dm]: It if finally mine!

Comment (RemieRichards) [inline code/modules/research/xenobiology/xenobiology.dm]: if -> is

Comment (Arianya) [inline code/modules/research/xenobiology/xenobiology.dm]: Whats proofreading? Who does that?

(Thanks Remie)

Comment (Armhulen) [inline code/modules/research/xenobiology/xenobiology.dm]: as if.

**Comment by Armhulen** [comment]
i don't like this lore, i don't like how you wrote it either

that>that>that
i liked the idea of giving creatures intelligence, not blocking some gaydar

**Comment by Arianya** [comment]
The issue arises from creatures not behaving as intelligent as they would normally be. Most noticeable with monkeys not ~~flinging poop~~ howling at you and trying to scrape your face off when you're jabbing them with needles.

There must be SOME explanation for this and a "gaydar" is the one that works best while also explaining how it works on non-regulated animals like mice.

**Comment by ShadowDeath6** [comment]
is this implying that all creatures were sentient at one point but nothing other than a sentience potion or literal magic will make them re-sentient?

**Comment by Arianya** [comment]
@ShadowDeath6 Consider intelligence of beings on the station to be divided into 3 levels:

sedate (i.e. simple_animal AI)

sentient (i.e. posessed by a player and able to move with intention and aims in mind)

human level intelligence (i.e. a biped who can speak and use opposable thumbs and throw cream pies)

The issue of "sentient" being being able to speak human language is one that I'll either tackle in another lore PR or would require removing that ability from non-bipeds entirely, and so would be a feature tweak in a lore PR which is a nono

**Comment by Iamgoofball** [comment]
tinfoil hats when

**Comment by Supermichael777** [comment]
M E R G E C O N F L I C T S

**Comment by Arianya** [comment]
@KorPhaeron Since it conflicts with your PR (#22014), do you object to this replacing your PR?

**Comment by KingPhilipIII** [comment]
All I can think about is the time as an op commander when I was saved by a sentient mulebot my underling used the potion on.

**Comment by TechnoAlchemisto** [comment]
Wow that is really wordy and bad

**Comment by Arianya** [comment]
@TechnoAlchemisto If you think this is wordy I pray you never see the Status Reports from Nanotrasen

**Comment by ExcessiveUseOfCobblestone** [comment]
>Wow that is really wordy and bad

**Comment by Armhulen** [comment]
you improved it so it's better, but the mad science sentience potion idea itself was cool

**Comment by Aloraydrel** [comment]
Not canon

**Comment by Arianya** [comment]
@Aloraydrel It includes plasma so its pretty much canon already :^)

>Wow that is really wordy and bad

Could you elaborate? Too long? Using too many fancy words where simple ones would suffice? What aspect of it was bad? I'd like to refine down what could be improved about it.

**Comment by TechnoAlchemisto** [comment]
You've overcomplicated something that already made sense and was good.

This makes so much less sense and is worse.

Creatures are creatures > given intelligence through dubious science

vs 

creatures always had intelligence of some sort > oppressed by some sort of fucking field > you produce electromagnetism to interrupt this field ???!

**Comment by Ausops** [comment]
this raises more lore questions than it answers

**Comment by Arianya** [comment]
@TechnoAlchemisto I can see what you mean in regards to the new changes, though I would object to calling the old method "dubious science" considering the lack of information in the old lore that it even was science as opposed to magical slime creating a potion out of thin air.

Will have a ponder on this today and revisit tonight. Will leave the PR open for now for further feedback since its not like its gonna get merged anyway.

**Comment by Grazyn** [comment]
I think that the message being too wordy and convoluted is actually an issue, since it draws attention away from the most important part which is that you're bound to serve the guy who created you. Not to mention that the message has an underlying tone of past oppression ("inhibiting field, sedation") that would imply that even though you have to serve the one guy, you could still hold a grudge against other NT personnel since they are responsible for keeping you dumb and imbecile.

**Comment by Arianya** [comment]
@Grazyn The subtle hostility is 100% intended, actually, as a hint/tip to the player that they should expect trouble from station crew, since sentienced animals are perma-valid and some fuckwit tends to want to bash them with a extinguisher.

Fair point re: wordy and convoluted, though the part about being bound is on a new line which should help? The whole thing needs revision anyway.

Slimes still behave notably differently pre and after sentience potion, so some level of sedation (albeit less effective then on normal animals, possibly as a result of them already being near-human-level intellect)

Gold slime hostile mobs are somewhat explained away by the use of plasma in their creation, as plasma is the key factor between this potion, the Sapience event lore and them. Blood and water gold slime mobs DO act sedate, atleast unless provoked some way.

Lavaland megafauna are excluded from the sentience potion, and normal mobs either go berserk or start fleeing (goldgrub) on seeing humanoids, but before that act sedate. Theres ways to handwave this distinction but admittedly this is another point that needs addressing.

**Comment by Grazyn** [comment]
Yeah I just don't like the whole "sedation" thing, it appears you thought of it with only monkeys in mind when every other mob has its own little life going on. For example, mice chew wires, cats and dogs have sex and can make (a lot of) babies, why wouldn't the field suppress these behaviours which are arguably quite annoying? Hostile mobs are also better at pathfinding towards their targets than your average human player, and can also dodge projectiles pretty fast. And plasma magic doesn't explain hostility from non-gold slime mobs like space carps, bears from centcomm, spiders etc.

All in all this field doesn't really look like it's working as advertised. And finally, the last question, WHY are these animals conscious and sentient in the first place? Your message only mentions that the field which was keeping them "sedated" no longer affects them, but it doesn't explain why they also gain human intelligence and the ability to talk. It means that the field was not only suppressing their animal-like behaviour, but their consciousness too. So all animals from pugs to goliaths are self-aware in their "normal" state? WHY? REEEE

**Comment by Arianya** [comment]
> Your message only mentions that the field which was keeping them "sedated" no longer affects them, but it doesn't explain why they also gain human intelligence and the ability to talk.

Refer to https://github.com/tgstation/tgstation/pull/22018#issuecomment-265600277

I agree though that there are too many plot holes at the moment for it to make sense, hence taking another crack tonight.

**Comment by ExcessiveUseOfCobblestone** [comment]
Why change it in the first place?

**Comment by optimumtact** [comment]
It seems people don't like this, so lets annoy cheridan for now.
